src/main/java/healthy/lifestyle/backend/mentals/repository/MentalJdbcRepository.java

```diff
@@ -0,0 +1,195 @@
+package healthy.lifestyle.backend.mentals.repository;
+
+import healthy.lifestyle.backend.mentals.model.Mental;
+import healthy.lifestyle.backend.mentals.model.MentalType;
+import healthy.lifestyle.backend.workout.repository.PageJdbc;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import lombok.*;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class MentalJdbcRepository {
```

Comment:
Why do you need all that Jdbc? I've found a better solution. If you copy-past code, I suggest you to pay attention is it a part of the production or at least testing release.